web/beacon-app/src/features/members/components/MemberDetailsPageLayout.tsx

```diff
@@ -0,0 +1,16 @@
+import { Route } from 'react-router-dom';
+
+import { PATH_DASHBOARD } from '@/application';
+import DashLayout from '@/components/layout/DashLayout';
+
+import MemberDetailsPage from './MemeberDetailsPage';
+
+export default function MemberDetailsPageLayout() {
+  return (
+    <>
+      <Route path={PATH_DASHBOARD.profile} element={<DashLayout />}>
+        <Route path={PATH_DASHBOARD.profile} element={<MemberDetailsPage />} />
+      </Route>
+    </>
+  );
+}
```

Comment:
@daniellemaxwell  can you check how we set up the router of the other pages please ?
in root :
```
      <Route element={<Root />} errorElement={<ErrorPage />}>
        <Route path="app" element={<PrivateRoute />}>
        <Route path="profile" element={<UserProfileComponent />} />
      </Route>  

```

Reply:
@masskoder Should this file be removed? https://github.com/rotationalio/ensign/blob/main/web/beacon-app/src/features/onboarding/components/Welcome/WelcomePageLayout.tsx

Reply:
yes it should be removed